packages/action-bar/stories/action-bar.stories.ts

```diff
@@ -13,69 +13,98 @@ governing permissions and limitations under the License.
 import { html, TemplateResult } from '@spectrum-web-components/base';
 
 import '@spectrum-web-components/action-bar/sp-action-bar.js';
-import '@spectrum-web-components/action-bar/sp-action-bar-scoped.js';
 import '@spectrum-web-components/field-label/sp-field-label.js';
 import '@spectrum-web-components/action-button/sp-action-button.js';
 import '@spectrum-web-components/action-group/sp-action-group.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-edit.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-more.js';
-
-import { ActionBarWrapper } from './';
+import { scopedStory } from '@spectrum-web-components/shared/src/scoped-story.js';
+import { ActionBarScoped } from '../src/ActionBarScoped.js';
+import { ActionButton } from '@spectrum-web-components/action-button';
+import { IconEdit } from '@spectrum-web-components/icons-workflow/src/elements/IconEdit.js';
+import { IconMore } from '@spectrum-web-components/icons-workflow/src/elements/IconMore.js';
+import { ScopedStory } from '@spectrum-web-components/shared';
 
 export default {
     title: 'Action Bar',
     component: 'sp-action-bar',
-    args: {
-        open: true,
-        emphasized: false,
-        variant: 'sticky',
-        flexible: false,
-        scoped: false,
-    },
 };
 
-type Properties = {
-    scoped?: boolean;
-    open?: boolean;
-    emphasized?: boolean;
-    variant?: 'sticky' | 'fixed';
-    flexible?: boolean;
-};
+/**
+ * scoped story creates a scoped component which assigns the tags used in the story template to be reused with scoped defination, however it would
```

Comment:
To create scoped stories we would need to send the definitions of children as well because when we create a scoped element it would need all definitions defined in  `ScopedRegistryHost` to be able to work properly.